### PR TITLE
Cart

### PR DESCRIPTION
**Pull request by SJTGSHIVAM**
Added: cart management 
Live Link: https://deploy-preview-5--log2-mock.netlify.app/

### CODE REVIEW

File: src/pages/Cart/cart.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  .lg2__aside--right {
+    grid-area: rside;
+  }
+  .lg2__footer {
+    grid-area: foot;
+  }
+  .lg2__card--cart {
+    grid-column: 2;
+    position: sticky;
+    grid-row: span 1git;
```
Comment (SJTGSHIVAM) [inline src/pages/Cart/cart.css]: I should remove this line, its not needed.

File: src/pages/Wishlist/Wishlist.tsx [hunk 2]
```diff
@@ -44,6 +44,7 @@ export const Wishlist = () => {
                     : false
                 }
                 invalidate={() => {
+                  console.log("its running!");
```
Comment (SJTGSHIVAM) [inline src/pages/Wishlist/Wishlist.tsx]: should remove this one as well.

File: src/App.tsx [hunk 3]
```diff
@@ -42,10 +59,7 @@ export default function App() {
           </Route>
           <Route path="/user" element={<ProtectedRoute />}>
             <Route path="" element={<h1>User page is under construction</h1>} />
```
Comment (tejas-muthya37) [inline src/App.tsx:61]: is the path missing something?

File: src/components/CartCard.tsx [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+      toastError();
+    }
+  };
+  const wishlistToggle = async (
+    id: string,
+    encodedToken: string,
+    invalidate: () => void
+  ) => {
+    try {
+      postUserWishlist(id, { authorinzation: encodedToken });
```
Comment (tejas-muthya37) [inline src/components/CartCard.tsx:81]: Typo here. *authorization.

File: src/components/Footer.tsx [hunk 5]
```diff
@@ -4,32 +4,32 @@ import {
...
-        </Link>
+        </a>
       </li>
       <li className="footerLink">
-        <Link to="https://www.linkedin.com/in/shivam-pandey-766524161">
+        <a
+          href="https://www.linkedin.com/in/shivam-pandey-766524161"
+          target="blank"
+        >
           <FaLinkedinIn />
```
Comment (tejas-muthya37) [inline src/components/Footer.tsx:26]: I think this might be <FaLinkedIn />

**Review by tejas-muthya37 — APPROVED** [review]
Looks good to merge. Liked your use of interface.

File: src/App.tsx (hunk 3 above)
Comment (SJTGSHIVAM) [reply src/App.tsx:61]: no its intentional

File: src/components/CartCard.tsx (hunk 4 above)
Comment (SJTGSHIVAM) [reply src/components/CartCard.tsx:81]: Hahaha this is really funny
this is spilled all over the code and this is why it works fine everywhere. will fix it in the next pr now.

Comment (SJTGSHIVAM) [reply src/components/CartCard.tsx:81]: I fixed it there

**Review by Hnh013 — APPROVED** [review]
good ui interface and functionality works accordingly , good to approve !

**Pull request merged**